content/departments/engineering/teams/code-intelligence/index.md

```diff
@@ -10,115 +12,161 @@ The Code Intelligence team builds tools and services that provide contextual inf
 
-## Tech stack
+The aim is for streams to have the following characteristics:
 
-Precise code intelligence specific services are written in Go and code intelligence features are added to the existing frontend service where possible. Sourcegraph extensions that provide code intelligence are written in TypeScript. We use Postgres databases to store data specific to a particular LSIF index and all other metadata.
+- Long lived, these are not ephemeral groupings.
+- Allow deep technical ownership and collaboration.
+- Building technical roadmaps with a view on the long term.
+- Firm but not brittle boundaries (people can operate across many streams when needed).
+- Allow us in the future to scale and create distinct teams with minimal overhead.
+- People feel 100% supported to comment, help and collaborate across the whole team.
 
-## How we do technical hiring
+There is no "that's not my responsibility" with the use of streams, we just have certain areas that are best filled by specific skill sets.
 
-- [Code Walkthrough](code-walkthrough.md)
+### Language Tools
 
-## Team communication
+- Slack channel: #language-tools-internal
+- Github board --> https://github.com/orgs/sourcegraph/projects/211/views/51
 
-Here are some key ways to contact us:
+##### Current members
 
-- On GitHub, mention or assign issues to [team/code-intelligence](https://github.com/sourcegraph/sourcegraph/issues/new?labels=team/code-intelligence)
-- On GitHub, view our [backlog](https://github.com/orgs/sourcegraph/projects/211) of work.
-- On Slack, use the #code-intel channel or tag the @codeintel team.
+- Olaf
+- TJ
+- Varun
 
-## Team process
+##### Focus and ownership
 
-The code intel team has defined a set of rituals that keeps the team connected around our common goals. As a team we also believe in good meeting etiquette, and to ensure effective meetings for everyone every meeting must have a meaningful topic, agenda, and stated goal or purpose statement defined ahead of time.
+The Language Tools stream focus on tooling specific to programming languages and their counterpart ecosystems to provide the best precise code navigation experience. The aim is to have a symbiotic relationship
+with the Language Platform team in that, Language Tools will provide the specific logic per problem (think syntax highlighting) whilst Language Platform will expose an agnostic/programatic way to scale this. Specific areas of ownership include:
 
-Our iterations last two weeks and are comprised of the following:
+- Precise Intelligence
+  - [SCIP](https://github.com/sourcegraph/scip#readme)
+  - [SCIP indexers](https://docs.sourcegraph.com/code_intelligence/references/indexers)
+  - LSIF indexers
+  - Community indexer contributions
+- Auto-inference behavior for indexing projects
+- Syntax highlighting (including the syntect service)
+- Package integrations (Maven,NPM, PyPI, Crates)
```

Comment:
```suggestion
- Package integrations (Maven, NPM, PyPI, Crates)
```

Reply:
(Apologies for pedantry again but:) It should be `npm` not NPM.